drivers/src/memory_layout.rs

```diff
@@ -67,6 +68,7 @@ pub const DCCM_SIZE: u32 = 256 * 1024;
 pub const ROM_DATA_SIZE: u32 = 996;
 pub const MAN1_SIZE: u32 = 17 * 1024;
 pub const MAN2_SIZE: u32 = 17 * 1024;
+pub const DATAVAULT_MAX_SIZE: u32 = 15 * 1024;
```

Comment:
15008 to be exact, looks like, but not sure if we need to align it to a multiple of 1024? (15008 is a multiple of 32 bytes.)

Reply:
We don't need to align to multiples of 1024. We'll cleanup these sizes in a targeted change.